nvm.sh

```diff
@@ -1789,7 +1789,7 @@ nvm_get_make_jobs() {
   elif [ "_$NVM_OS" = "_sunos" ]; then
     NVM_CPU_THREADS="$(psrinfo | wc -l)"
   elif [ "_$NVM_OS" = "_aix" ]; then
-    NVM_CPU_THREADS="$(lsconf | command grep 'Number Of Processors:'| command awk '{print $4}')"
+    NVM_CPU_THREADS="$(pmcycles -m | wc -l)"
```

Comment:
@ljharb As I understand it, `pmcycles -m` gives you the number of CPU cores, so this is actually the right change.